src/client/componentsMealsharing/PostForm.js

```diff
@@ -0,0 +1,85 @@
+import React,{Component,useState} from "react";
+
+export function PostForm() {
+    const [seat, setSeat] = useState("");
+    const [phone, setPhone] = useState("");
+    const [name, setName] = useState("");
+    const [mail, setMail] = useState("");
+    const [reservation, setReservation] = useState([]);
+    bookSeats = ()=> {
+       setReservation (prev=>{[
+         ...prev,
+         {
+           number_of_guests: seat,
+           // created_date: request.body.created_date,
+           contact_phonenumber: phone,
+           contact_name: name,
+           contact_email: mail,
+           // meal_id: request.body.meal_id,
+           // number_of_guests: seat,
+         },
+       ];})
+
+       const options = {
+         method: "POST",
+         headers: {
+           "Content-Type": "application/json",
+         },
+         body: JSON.stringify(data)
```

Comment:
here i can not see the  data obj that you need to post it to the database

Reply:
Yea the variable `data` is undefined